src/ImageSharp/Processing/Processors/Dithering/OrderedDither.cs

```diff
@@ -23,7 +22,7 @@ namespace SixLabors.ImageSharp.Processing.Processors.Dithering
         /// Initializes a new instance of the <see cref="OrderedDither"/> struct.
         /// </summary>
         /// <param name="length">The length of the matrix sides</param>
-        [MethodImpl(InliningOptions.ShortMethod)]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This method does not seem trivial. I would keep it as `ShortMethod` so it shows up in profiler results when asked for.

Reply:
Happy to revert